scriptworker/task.py

```diff
@@ -240,6 +243,25 @@ def get_and_check_tasks_for(task, msg_prefix=''):
     return tasks_for
 
 
+# get_repo_scope {{{1
+def get_repo_scope(task):
+    """Given a parent task, return the repo scope for the task.
+
+    Background in https://bugzilla.mozilla.org/show_bug.cgi?id=1459705#c3
+
+    Args:
+        task (dict): the task definition.
+
+    Returns:
+        str: the ``repo_scope``
+        None: if no ``repo_scope`` is found
+
+    """
+    for scope in task['scopes']:
+        if REPO_SCOPE_REGEX.match(scope):
+            return scope
```

Comment:
I'm afraid this might cover an issue. Let's imaging we have these 2 scopes:
```
assume:repo:some-repo:action:action1
assume:repo:some-repo:action:action2 
```

The first repo scope will always be returned, even though we might want the `action2` scope. Am I getting this right?

Reply:
That's correct. If the correct `repo_scope` is actually `assume:repo:some-repo:action:action2`, then the json-e render will create a mismatched task definition, and CoT verification will fail.

At that point we'll hopefully realize that having two `assume:repo:some-repo:action:*` scopes in the same task is a bad idea, and remove one.

Reply:
Would you prefer we die on multiple `repo_scope`s ?

We're getting rid of `repo_scope` at some point (I think when all actions are hooks), so I don't want to require `repo_scope` not be `None`.

Reply:
Yeah, dying on multiple scopes sounds safer too me. That would immediately shows something broke.

Reply:
Ok, added that check.